### PR TITLE
DT-911: Add in services dashboard

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql [hunk 1]
```diff
@@ -49,8 +49,8 @@ VALUES ('BOOK_MOVE', 'Book a secure move', 'Book a secure move', 'ROLE_PECS_SUPP
        ('POM', 'Allocate a POM Service', 'Allocate the appropriate offender manager to a prisoner', 'ROLE_ALLOC_MGR', 'https://moic.service.justice.gov.uk', 1, 'https://moic.service.justice.gov.uk/help'),
        ('PATHFINDER', 'Pathfinder Service', 'View and Manage Pathfinder nominals', 'ROLE_PF_STD_PRISON,ROLE_PF_APPROVAL,ROLE_PF_POLICE', 'http://localhost:3000', 1, null),
        ('SOC', 'Manage SOC cases', 'View and manage SOC cases', 'ROLE_SOC_CUSTODY,ROLE_SOC_COMMUNITY,ROLE_SOC_RESTRICTED', 'http://localhost:3000', 1, null),
-       ('USERADMIN', 'Admin & Utilities Service', 'Admin & utilities Service For NOMIS and Auth User', 'ROLE_KW_MIGRATION,ROLE_MAINTAIN_ACCESS_ROLES,ROLE_MAINTAIN_ACCESS_ROLES_ADMIN,ROLE_MAINTAIN_OAUTH_USERS,ROLE_AUTH_GROUP_MANAGER', 'http://localhost:3001/admin-utilities', 1, null),
-       ('DETAILS', 'Manage account details', 'View and change your account details', null, '/auth/account-details', 1, null);
+       ('USERADMIN', 'M<anage user accounts', null, 'ROLE_KW_MIGRATION,ROLE_MAINTAIN_ACCESS_ROLES,ROLE_MAINTAIN_ACCESS_ROLES_ADMIN,ROLE_MAINTAIN_OAUTH_USERS,ROLE_AUTH_GROUP_MANAGER', 'http://localhost:3001/admin-utilities', 1, null),
```
Comment (petergphillips) [inline src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql:52]: just to keep consistent with dev etc.

File: src/main/resources/templates/ui/services.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  <div sec:authorize="hasRole('ROLE_OAUTH_ADMIN')">
+    <h2 class="govuk-heading-m">Services</h2>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col">Name</th>
+        <th class="govuk-table__header" scope="col">Description</th>
+        <th class="govuk-table__header" scope="col">Roles</th>
+        <th class="govuk-table__header" scope="col">Url / email</th>
```
Comment (petergphillips) [inline src/main/resources/templates/ui/services.html:20]: url and email were taking up too much space as separate columns

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ServicesSpecification.kt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import org.junit.jupiter.api.Test
+import org.openqa.selenium.support.FindBy
+
+
+class ServicesSpecification : AbstractAuthSpecification() {
+  @Page
+  private lateinit var servicesSummaryPage: ServicesSummaryPage
+//
+//  @Page
+//  private lateinit var servicesMaintenancePage: ServicesMaintenancePage
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ServicesSpecification.kt:17]: commented out, will add back in when edit completed

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/ClientControllerTest.kt [hunk 4]
```diff
@@ -23,10 +21,9 @@ import org.springframework.ui.ExtendedModelMap
 import uk.gov.justice.digital.hmpps.oauth2server.security.AuthSource
 import uk.gov.justice.digital.hmpps.oauth2server.security.UserDetailsImpl
 
-@ExtendWith(MockitoExtension::class)
 class ClientControllerTest {
   private val clientDetailsService: JdbcClientDetailsService = mock()
-  private val telemetryClient: TelemetryClient = com.nhaarman.mockito_kotlin.mock()
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/ClientControllerTest.kt:29]: just a tidy up

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql (hunk 1 above)
Comment (Mjwillis) [reply src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql:52]: Typo?

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ServicesSpecification.kt [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      Allocate the appropriate offender manager to a prisoner 
+      [ROLE_ALLOC_MGR] 
+      https://moic.service.justice.gov.uk / https://moic.service.justice.gov.uk/help 
+      true
+ """.replaceIndent().replace("\n", ""))
+    return this
+  }
+
+//  fun editService(service: String = "apireporting") {
+//    el("#edit-$service").click()
```
Comment (Mjwillis) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ServicesSpecification.kt:75]: Old code?

**Pull request merged**